.azure-pipelines/apiview.yml

```diff
@@ -165,11 +165,10 @@ stages:
               pwsh: true
             displayName: 'Start Azurite'
 
-          - task: UseDotNet@2
-            displayName: 'Install .NET'
-            inputs:
-              version: '$(DotNetRuntimeVersion)'
-          
+          - template: /eng/pipelines/templates/steps/install-dotnet.yml
+            parameters:
+              DotNetCoreVersion: $(DotNetRuntimeVersion)
```

Comment:
Do we need to pass this here given it aligns with the default? We should consider removing the DotNetRuntimeVersion property completely. 

cc @praveenkuttappan

Reply:
@weshaggard @praveenkuttappan the variable is [hardcoded to 6.x](https://github.com/Azure/azure-sdk-tools/blob/c1fcc8f3687828009bf5653bb225752f80fe85c4/.azure-pipelines/apiview.yml#L32). It is used in two places: here, and for [PublishTestResult@2 / TestRunTitle](https://github.com/Azure/azure-sdk-tools/blob/c1fcc8f3687828009bf5653bb225752f80fe85c4/.azure-pipelines/apiview.yml#L206):  

``` yml
- task: PublishTestResults@2
  condition: succeededOrFailed()
  inputs:
    testResultsFiles: '**/*.trx'
    testRunTitle: 'Windows DotNet $(DotNetRuntimeVersion)'
    testResultsFormat: 'VSTest'
    mergeTestResults: true
```

Removing the variable would probably necessitate removing the .NET version from the `testRunTitle`, unless you folks have some idea how to pull it into this file out from [`install-dotnet.yml`](https://github.com/Azure/azure-sdk-tools/blob/5101537df19babeffbb21eb00843f25cb0b018a4/eng/pipelines/templates/steps/install-dotnet.yml) or from [`global.json`](https://github.com/Azure/azure-sdk-tools/blob/5101537df19babeffbb21eb00843f25cb0b018a4/global.json).

But I guess it is not a big loss if we remove the version from here?

Reply:
OK I went ahead [and removed the version variable](https://github.com/Azure/azure-sdk-tools/pull/5045/commits/baa32b74a6d1ec2211e5809bc59c5e4603d11a7d).

Reply:
Yes I would suggest to just remove the version from the title.